### PR TITLE
Feature/fbar 147 filter bar remove submit

### PR DESCRIPTION
**Pull request by paulmskim**
**Ticket:** [FBAR-147]

This PR tweaks the loader to be more accessible and adds the `aria-busy` attribute to the container.

**Screencast:** https://drive.google.com/file/d/1GJmrj6D4RbYJ2YyhlLg9MXjw653v9iQ3/view?usp=sharing

[FBAR-147]: https://moderntribe.atlassian.net/browse/FBAR-147

### CODE REVIEW

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 2 error(s)

File: src/views/v2/components/loader.php [hunk 1]
```diff
@@ -9,15 +9,18 @@
  *
  * @link http://m.tri.be/1aiy
  *
- * @version 4.9.10
+ * @version TBD
  *
  */
```
Comment (tr1b0t) [inline src/views/v2/components/loader.php:14]: * :no_entry_sign: There must be exactly one blank line after the file comment
* :no_entry_sign: Additional blank lines found at end of doc comment

<sub>hash: c4cdd0698143012fbb5d89f4476a563b</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 2 error(s)

File: src/views/v2/components/loader.php [hunk 2]
```diff
@@ -9,15 +9,17 @@
  *
  * @link http://m.tri.be/1aiy
  *
- * @version 4.9.10
- *
+ * @version TBD
  */
 ?>
```
Comment (tr1b0t) [inline src/views/v2/components/loader.php:14]: * :no_entry_sign: There must be exactly one blank line after the file comment
* :no_entry_sign: Additional blank lines found at end of doc comment

<sub>hash: c662e397c07ee2031214e56a5b02f8eb</sub>

**Review by lucatume — APPROVED** [review]
Fix the smaller style issues, then good to go.

**Comment by paulmskim** [comment]
@lucatume pinged G about this, not sure what's going on with the sniffer here. I've tried various things but not working. We're going to skip this.

**Comment by lucatume** [comment]
> @lucatume pinged G about this, not sure what's going on with the sniffer here. I've tried various things but not working. We're going to skip this.

Whatever, merge away and skip it, we _should_ not die for a missing blank line.

**Pull request merged**